app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -974,15 +976,13 @@ XrResult OpenXRInputSource::UpdateInteractionProfile(ControllerDelegate& delegat
         }
     }
 
+    uint32_t numHaptics = 0;
```

Comment:
These changes seems unrelated to the new logic added, for allowing Wolvic operating without any active mapping (eg, handtracking).

Reply:
They are actually related because if there are not mappings then the haptic count is never set (even if it's zero)